code/package.json

```diff
@@ -35,6 +35,7 @@
     "karma-coverage": "~2.2.0",
     "karma-jasmine": "~5.1.0",
     "karma-jasmine-html-reporter": "~2.1.0",
+    "prettier": "^3.4.2",
```

Comment:
`prettier` was nowhere to be found. This will make sure that we 1) use prettier, 2) user prettier with the same default settings.